src/common/sound.cpp

```diff
@@ -14,6 +14,34 @@ Sound::Sound() {
     frequency = 100.f; // frequency of sound signal (0-1000)
     volume = 0.50;     // volume of sound signal (0-1)
 
+    // -- values of sound signals - frequencies, volumes, durations
+    // -- durations of beep
+    durations[eSOUND_TYPE_ButtonEcho] = 100.f;
+    durations[eSOUND_TYPE_StandardPrompt] = 500.f;
```

Comment:
oh, durations are uin32_t, why do you assign floating point numbers?

Reply:
my mistake - i copied previous frequency values and compiler did not give a damn about that.